components/multi-platform-controller/production-downstream/stone-prod-p02/host-values.yaml

```diff
@@ -11,7 +11,7 @@ archDefaults:
     key-name: "konflux-prod-int-mab01"
     security-group-id: "sg-0903aedd465be979e"
     subnet-id: "subnet-02c476f8d2a4ae05e"
-  windows-amd64:
+  windows-4xlarge-amd64:
```

Comment:
Correct me if I'm wrong but if we change this value here, we'd need to change also the template in `components/multi-platform-controller/base/host-config-chart/templates/host-config.yaml` as, there, we read the default values for each platform and it should match what we define here; it looks like this as of this commit:
```
  {{- $windows := (index .Values "archDefaults" "windows") | default (dict) }}
```
 otherwise, when running `kustomize build, we'll have non explicitly defined fields with a null value.